pyproject.toml

```diff
@@ -28,11 +28,13 @@ url = "https://pypi.rasa.com/simple/"
 [tool.towncrier]
 package = "rasa"
 package_dir = "rasa"
-filename = "CHANGELOG.rst"
+filename = "./newdocs/docs/changelog.mdx"
```

Comment:
I sometimes find it useful to have the changelog accessible at the root of the repository - some members of our community might feel the same? If that's the case, we could definitely keep the same setup (changelog at root + include the changelog file in the docs)

Reply:
:+1: - I've added a symlink in the root directory

Reply:
nvm, doesn't work properly with the github ui 😞

Reply:
arf - the symlink the other way around then I guess?

Reply:
yes fingers crossed the javascript build system properly follows them

Reply:
just tried this, doesn't work either. webpack doesn't properly resolve the symlink. I think we need to work without this (or add a file that just contains a link to the changelog on the webpage, ala
`You can find the changelog at https://rasa.com/docs/changelog`)

Reply:
@tmbo would [this](https://webpack.js.org/configuration/resolve/#resolvesymlinks) work?

Reply:
I did find this but I didn't try anything there (assumed that `boolean = true` means `true` is the default). But might be worth a try. Can't look into this though right now so if you have time to try, feel free to give it a go

Reply:
yes, sure will do if I can

Reply:
I think it's some sort of protection from docusaurus. It can be easily fixed by copying the changelog at "pre build" time, a bit like what I did for the variables or the program-output directive.

(but the thing I learned is that [mdx supports transclusion](https://mdxjs.com/getting-started#documents)!)

Reply:
nice :+1:

Reply:
@tmbo 968e4a1b418

Reply:
great, thanks a lot for looking into moving the file to the root :star: 

should we resolve this thread? don't think there is anything left

Reply:
yep, doing it 👍